whole-tale/ngx-dashboard
# fix: shorten rd by omitting hostname

Companion to https://github.com/whole-tale/girder_wholetale/pull/552 . Shorten the redirect url by not hardcoding dashboard's hostname. 

## How to Test

1. Login to dashboard.
2. Create a Tale, make it public. Navigate to view / interact tab.
3. Copy the url.
4. Logout.
5. In an incognito session: open the url from 3.
6. You should end up being forced to login and afterwards on the proper interact tab.
